Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -1,570 +1,588 @@
-/* Style applied to the TreeView. */
-
 ul.treeViewRoot
 {
+  --treeview-chevron-size: 1em;
+  --treeview-chevron-gap: 0.5em;
+  --treeview-padding: calc(var(--treeview-chevron-size) + var(--treeview-chevron-gap));
+
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
-  list-style: none;
 }
 
-/* Applied to the top level node's children area if the expandre is hidden for the top level nodes. */
-ul.treeViewTopLevelNodeChildren
+ul.treeViewRoot a
 {
-  margin: 0;
-  padding: 0;
-  list-style: none;
+  color: black;
+  text-decoration: none;
 }
 
-/* Applied to the node's children area. */
-ul.treeViewNodeChildren
+ul.treeViewRoot .treeViewNode
 {
-  background-image: url(../Image/sprite.svg#TreeViewI);
-  background-size: 16px 16px;
-  background-repeat: repeat-y;
-  margin: 0;
-  padding: 0;
-  list-style: none;
+  position: relative;
 }
 
-/* Applied to the node's children area if the nodes should not be connected by lines. */
-ul.treeViewNodeChildrenNoLines
+ul.treeViewRoot,
+ul.treeViewRoot ul.treeViewNodeChildren,
+ul.treeViewRoot ul.treeViewNodeChildrenNoLines,
+ul.treeViewRoot ul.treeViewTopLevelNodeChildren
 {
-  margin: 0;
   padding: 0;
   list-style: none;
 }
 
-ul.treeViewRoot li
+ul.treeViewRoot span.treeViewNodeHeadSelected a:focus,
+ul.treeViewRoot li[role=treeitem]
 {
   outline: none;
 }
 
-ul.treeViewRoot > li,
-ul.treeViewTopLevelNodeChildren > li
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHead a > span,
+ul.treeViewRoot li:focus > span.treeViewNode span.treeViewNodeHead a:hover > span
 {
+  text-decoration: underline;
 }
 
-ul.treeViewNodeChildren > li,
-ul.treeViewNodeChildrenNoLines > li
+ul.treeViewRoot span.treeViewNode:hover .treeViewNodeHead:before,
```

Comment:
Hover highlight doesn't work starting where the absolute positon is greater than the current viewport.
![image](https://user-images.githubusercontent.com/6972548/132822241-e9246d38-979a-4ea1-9aa0-1303bdd4d58c.png)